components/edit-collective/sections/Tiers.js

```diff
@@ -399,6 +399,7 @@ class Tiers extends React.Component {
     const { intl, collective, defaultType = 'TICKET' } = this.props;
     const hasCustomContributionsDisabled = get(collective, 'settings.disableCustomContributions', false);
     const isNew = !collective.id;
+    const displayCustomContributionsSettings = !isNew && defaultType !== 'TICKET';
```

Comment:
IsNew variable is never used and I guess we can get rid of it by directly referencing collective.id